src/analyses/variable-sensitivity/monotonic_change.cpp

```diff
@@ -0,0 +1,331 @@
+/*******************************************************************\
+
+ Module: Predicate abstraction for monotonic change
+
+ Authors: Long Pham, Saswat Padhi
+
+\*******************************************************************/
+
+// For the documentation of the class monotonic_changet, have a look at
+// the header file monotonic_change.h.
+
+#include <langapi/language_util.h>
+
+#include <util/interval.h>
+#include <util/std_expr.h>
+
+#include "abstract_object_statistics.h"
+#include "monotonic_change.h"
+
+// We will print out a message on the standard output when we encouter a
+// pathological case. An example is when we are try to merge the abstract value
+// "uninitialized" with other abstract values (e.g. "increase"). This is why we
+// need <iostream>.
+#include <iostream>
```

Comment:
Should we use `log` functions instead of `cout`?

@martin-cs May be you could advise us?